mods/ra2/weapons/misc.yaml

```diff
@@ -69,3 +69,11 @@ PsiWave:
 		DamageTypes: PsychicDeath
 	Warhead@2Eff: CreateEffect
 		Explosions: psi_wave
+
+CRPsiWave:
+	Inherits: PsiWave
+	ReloadDelay: 20
+	Projectile: Missile
+		RangeLimit: 0c1
```

Comment:
> Error: Weapon `crpsiwave`: projectile RangeLimit lower than weapon range!

Reply:
Since when this produces an error?! That's stupid.

Reply:
Ahwell, that means OpenRA needs my InstantExplode projectile longterm. How unsurprising.

Reply:
Whatever, i'll drop the Yuri IFV, this is blocking Auto Target PR and it is more important.